src/pulselistener/tests/conftest.py

```diff
@@ -19,6 +26,37 @@ def status(self, task_id):
                     }
             assert False
 
+        def task(self, task_id):
+            now = datetime.utcnow()
+
+            if 'retry:' in task_id:
+                retry = int(task_id[task_id.index('retry:')+6])
```

Comment:
Surprised that flake8 did not complain here, it usually complains about no space before/after the math operator.